mlrun/api/api/endpoints/utils.py

```diff
@@ -0,0 +1,14 @@
+import re
+from hashlib import sha1
+
+# uid is hexdigest of sha1 value, which is double the digest size due to hex encoding
+hash_len = sha1().digest_size * 2
+uid_regex = re.compile("^[0-9a-f]{{{}}}$".format(hash_len), re.IGNORECASE)
```

Comment:
Prefer to use python3 string formatting
```suggestion
uid_regex = re.compile(f"^[0-9a-f]{{{hash_len}}}$", re.IGNORECASE)
```

Reply:
Done.